src/archivist/recorder/repositories/mongo/index.js

```diff
@@ -24,6 +24,8 @@ export default class MongoRepository extends RepositoryInterface {
 
     this.collection = db.collection(this.collectionName);
 
+    await this.collection.createIndex({ serviceId: 1, termsType: 1, fetchDate: -1 });
```

Comment:
Do we really want to block with `await`? 🤔

Reply:
Not necessary